docs/sql-guide/functions/function-totimestamp.md

```diff
@@ -57,17 +58,20 @@ insert into demo(_id, ts)
     values (4, -86400);
 
 select _id, ts from demo;
-
- _id | ts                            
------+-------------------------------
-   1 | 1970-01-01 00:00:00 +0000 UTC 
-   2 | 1970-01-02 00:00:00 +0000 UTC 
-   3 | 1970-01-02 01:01:01 +0000 UTC 
-   4 | 1969-12-31 00:00:00 +0000 UTC 
 ```
 
-### Convert granular integers to timestamp.
-To convert integer date-time values with non-standard granularity TOTIMESTAMP() can be used with appropriate `timeunit` parameter that matches the granularity of the input. 
+Result:
+
+| _id | ts |
```

Comment:
I'm actually a little worried about doing this because the majority of examples are not returned as tables, but as the natural output of fbsql. Converting results into tables can take a lot of time, especially for result sets with many columns. Thoughts? Were you planning to do this for all examples?

Reply:
Natural output wins.

So would it look like the original content I turned into a table? Or something else?

Reply:
Ya the original content I believe. the majority of our sql function examples just has the natural output in the code block: https://docs.featurebase.com/docs/sql-guide/functions/function-prefix/